projects/1.20/assets/productivetrees/productivetrees/patchouli_books/guide/zh_cn/entries/guide/start.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "初期准备",
+    "icon": "minecraft:writable_book",
+    "category": "productivetrees:guide",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "flag": "mod:productivebees",
+            "text": "首先，你需要在蜜蜂附近并排种植两棵树木。$(br)蜂箱可以是虚拟的，每一个安装在蜂箱的范围升级可以使蜜蜂的工作范围延展2格。基础范围是4格。$(br2)需注意的是，模组仍在开发中，部分特性尚未完成。$(br)尤其是一些树木（生长后的外观）有待改进。"
```

```suggestion
            "text": "首先，你需要在蜜蜂附近并排种植两棵树木。$(br)可以使用模拟蜂箱，每一个安装在蜂箱的范围升级可以使蜜蜂的工作范围延展 2 格。基础范围是 4 格。$(br2)需注意的是，模组仍在开发中，部分特性尚未完成。$(br)尤其是一些树木（生长后的外观）有待改进。"
```
帕秋莉中的全半角字符间应该还是要空格的
~~虽然也不知道是否真要这么做才和mc本体的视觉效果一致~~